crates/rspack_core/src/tree_shaking/visitor.rs

```diff
@@ -48,27 +48,29 @@ pub enum SymbolRef {
 impl SymbolRef {
   pub fn src(&self) -> ModuleIdentifier {
     match self {
-      SymbolRef::Direct(d) => d.src(),
+      SymbolRef::Declaration(d) => d.src(),
       SymbolRef::Indirect(i) => i.src(),
       SymbolRef::Star(s) => s.src(),
       SymbolRef::Url { src, .. } => *src,
+      SymbolRef::Usage(_, _, src) => *src,
     }
   }
 
   pub fn importer(&self) -> ModuleIdentifier {
     match self {
-      SymbolRef::Direct(d) => d.src(),
+      SymbolRef::Declaration(d) => d.src(),
       SymbolRef::Indirect(i) => i.importer,
       SymbolRef::Star(s) => s.module_ident(),
       SymbolRef::Url { importer, .. } => *importer,
+      SymbolRef::Usage(_, _, src) => *src,
     }
   }
   /// Returns `true` if the symbol ref is [`Direct`].
   ///
   /// [`Direct`]: SymbolRef::Direct
```

Comment:
Documentation should be updated as well correct?

Reply:
see https://github.com/web-infra-dev/rspack/pull/3755